src/hooks/context/hook.tsx

```diff
@@ -0,0 +1,15 @@
+import { container } from 'tsyringe';
+
+import { ContextService } from './contextService';
+import { ContextFile } from './models';
+
+export function useContextFile() {
+	const contextService: ContextService = container.resolve(ContextService);
+
+	return {
+		list: () => {
```

Comment:
I would segregate this, make a list method that transform the storaged data to make it ready to use on the ListContextComponent and one method that transform the necessary data to make it ready to use on the ShowCurrentContextComponent 🤔